### ISSUE TITLE
Use of custom model for Anomaly detection

### ISSUE THREAD

**Issue by ayanray089** [hastic/hastic-server#887]
Hi,

I have a custom ML model for anomaly detection of time series data which has been already trained. Is it possible to use that custom ML model for predicting the Anomalous time windows after the user labels a time window, instead of default algorithms which hastic uses. I want the results to be displayed in grafana through hastic-grafana panel.
Reference to a working example will help.

Thanks,
Ayan

**Comment by jonyrock** [comment]
@ayanray089 yes it's possible. I supposed that you have ssen this https://github.com/hastic/hastic-server/wiki/Custom-model
But be prepared that it's not going to be easy.

**Comment by ayanray089** [comment]
Thanks!

Do you have any working example?

Thanks,
Ayan

**Comment by jonyrock** [comment]
@ayanray089 please write email if you really want to go through this:) ping@hastic.io
